projects/ngrx.io/content/guide/component-store/effect.md

```diff
@@ -71,3 +71,21 @@ export class MovieComponent {
 
 }
 </code-example>
+
+## tapResponse
+An easy way to handle the response in ComponentStore effects in a safe way, without additional boilerplate is to use the `tapResponse` operator. It enforces that the error case is handled and that the effect would still be running should an error occur. It is essentially a simple wrapper around `tap` that handles success and error + `catchError(() => EMPTY)`. 
```

Comment:
The `catchError(() => EMPTY)` is confusing me a little here,  is it missing something or is it just me 😅

Reply:
```suggestion
An easy way to handle the response in ComponentStore effects in a safe way, without additional boilerplate is to use the `tapResponse` operator. It enforces that the error case is handled and that the effect would still be running should an error occur. It is essentially a simple wrapper around two operators:
- `tap` that handles success and error
- `catchError(() => EMPTY)` that ensures that the effect continues to run after the error.
```

@timdeschryver Maybe something like this?

Reply:
Yes, I think that's better. Thanks @alex-okrushko